Remove all images

TomMaullin opened the pull request:
This is a very quick PR to address issue #27 (`When re-running a WB analysis, not all images are removed`) - please let me know if you have any feedback/suggestions!

TomMaullin commented:
Hi @nicholst ,

This PR is now review ready and all tests pass! It's only a small PR with a simple fix but please let me know if you have any feedback/suggestions!

nicholst commented:
Oh! Killing everything swe_* seems dangerous. Eg what if someone had
created a file “swe_notes.txt” which seems like an unwanted (& surprising)
artifact.

Can you implement the more specific version I proposed (but catching all
possible files)?
-- 
_________________________________________________________
Thomas Nichols, PhD
Professor of Neuroimaging Statistics
Nuffield Department of Population Health | University of Oxford
Big Data Institute | Li Ka Shing Centre for Health Information and Discovery
Old Road Campus | Headington | Oxford | OX3 7LF | United Kingdom
T: +44 1865 743590 | E: thomas.nichols@bdi.ox.ac.uk
W: http://nisox.org | http://www.bdi.ox.ac.uk

TomMaullin commented:
Hi @nicholst ,

Ah okay - yes sure. The reason I had deleted all `swe_*` files was as it may be more robust to any future filename changes but I understand the concern. I have now changed it to list all possible file title formats. Please let me know if you have any further feedback!
